string node

TopicInfo[] publications
TopicInfo[] subscriptions
string[] services

